src/pages/public_registration/PublicRegistration.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { FC } from 'react';
+import { CircularProgress } from '@material-ui/core';
+
+import { RegistrationStatus } from '../../types/registration.types';
+import { useParams } from 'react-router-dom';
+import useFetchRegistration from '../../utils/hooks/useFetchRegistration';
+import PublicRegistrationContent from './PublicRegistrationContent';
+import NotPublished from '../errorpages/NotPublished';
+import NotFound from '../errorpages/NotFound';
+import { useSelector } from 'react-redux';
+import { RootStore } from '../../redux/reducers/rootReducer';
+
+const PublicRegistration: FC = () => {
+  const { identifier } = useParams();
+  const [publication, isLoadingPublication] = useFetchRegistration(identifier);
+  const user = useSelector((store: RootStore) => store.user);
+
+  const isAllowedToSeePublicPublication =
+    publication?.status === RegistrationStatus.PUBLISHED || user?.isCurator || publication?.owner === user?.id;
```

Comment:
```suggestion
    publication?.status === RegistrationStatus.PUBLISHED || (user?.isCurator && publication.publisher.id === user?.id) || publication?.owner === user?.id;
```
vil tro Curator bare skal kunne forhåndsvise *registreringer* for egen institusjon, men det er antagelser, så gjør hva du vil med det :)

Reply:
du har nok helt rett i det ja - kan godt ta det med (selv om det ikke har noe med temaet for PRen å gjøre), men er jo så lite, så går nok bræ :)